doc/howtos/rdtraining/15_qwebintro.rst

```diff
@@ -0,0 +1,133 @@
+.. _howto/rdtraining/15_qwebintro:
+
+================================
+Part 15: A Brief History Of QWeb
+================================
+
+Up to now, the design of the interface of our real estate module was rather limited. Building
+a list view is straightforward since only the list of fields is necessary. The same holds true
+for the form view: despite the use of a few tags such as ``<group>`` or ``<page>``, there
+is very little to do in terms of design.
+
+However, if we want to give a unique look to our application, it is necessary to go a step
+further and be able to design new views. Moreover, other features such as PDF reports or
+website pages need another tool to be created with more flexibility: a templating_ engine.
+
+You might already be familiar with existing engines such as Jinja (Python), ERB (Ruby) or
+Twig (PHP). Odoo comes with its own built-in engine: :ref:`reference/qweb`.
+QWeb is the primary templating engine used by Odoo. It is an XML templating engine and used
+mostly to generate HTML fragments and pages.
+
+You probably already came across the `kanban board`_ in Odoo: the records are displayed in a
+card-like structure. We will build such a view for our real estate module.
+
+Concrete Example: A Kanban View
+===============================
+
+**Reference**: the documentation related to this topic can be found in
+:ref:`reference/views/kanban`.
+
+.. note::
+
+    **Goal**: at the end of this section a Kanban view of the properties is created:
+
+    .. image:: 15_qwebintro/media/kanban.png
+        :align: center
+        :alt: Kanban view
+
+In our estate application, we would like to add a Kanban view to display our properties. Kanban
+views are standard Odoo views (such as the form or list view), but their structure is much more
+flexible. In fact, the structure of each card is a mix of form elements (including basic HTML)
+and QWeb. The definition of a Kanban view is similar to the definition of the list and form
+views, except that their root element is ``<kanban>``. In its simplest form, a Kanban view
+looks like:
+
+.. code-block:: xml
+
+    <kanban>
+        <templates>
+            <t t-name="kanban-box">
+                <div class="oe_kanban_global_click">
+                    <field name="name"/>
+                </div>
+            </t>
+        </templates>
+    </kanban>
+
+Let's break down this example:
+
+- ``<templates>``: defines a list of :ref:`reference/qweb` templates. Kanban views *must* define at
+  least one root template ``kanban-box``, which will be rendered once for each record.
```

Comment:
Explain what other templates we'd define? Maybe show / link to an ornate kanban which uses multiple templates?